graylog2-server/src/main/java/org/graylog2/bindings/providers/JestClientProvider.java

```diff
@@ -70,6 +74,10 @@ public JestClientProvider(@Named("elasticsearch_hosts") List<URI> elasticsearchH
                             new AuthScope(hostUri.getHost(), hostUri.getPort(), AuthScope.ANY_REALM, hostUri.getScheme()),
                             new UsernamePasswordCredentials(username, password)
                         );
+
+                        if (!Strings.isNullOrEmpty(username) || !Strings.isNullOrEmpty(password)) {
```

Comment:
No, both `username` and `password` could be empty, so we add it if at least one of them is present. Admittedly, using _only_ `username` or `password` for authentication is seldom, but still valid and possible.